### DISCUSSION TITLE
Downloaded page

### DISCUSSION THREAD

**Discussion by farukara** [jeff-hughes/shellcaster#40] [Q&A]
Is there a "**downloaded episodes**" section/page so that I can see all downloaded episodes in one place, instead of hunting down separately?

**Comment by jeff-hughes** [comment]
You know, there isn't currently a feature like that, but I can see that being useful. I think something equivalent that might be a bit easier to implement would be a couple of "filters" you can toggle on and off to only show downloaded episodes (or only non-downloaded ones, or all), or only show unplayed episodes (or only played, or all). That way, it keeps the same layout in the same window, it just hides ones you're not wanting to show.

I'll add it to my list! Thanks for bringing up the idea :)
